_includes/live-demos/rtc/index.js

```diff
@@ -0,0 +1,320 @@
+/* The following functions are used for randomizing data in this demo */
+function getRandomInt(max) {
+  /* Random number generator */
+  return Math.floor(Math.random() * max);
+}
+
+/* Used for selecting two users at random */
+function getTwoRandomInt(max) {
+  /* Generates two random numbers (with removal). */
+  let tempArray = [...Array(max).keys()];
+  const valueOne = getRandomInt(tempArray.length);
+  tempArray = tempArray.filter((item) => item !== valueOne);
+  const valueTwo = tempArray[getRandomInt(tempArray.length)];
+  return [valueOne, valueTwo];
+}
+
+function randomString() {
+  /*
+    * Generating a random string to use as a document ID, Encryption key, and
+    * Encryption keyHint (randomized for demonstration purposes only).
+    * These values can be any string, but must be unique.
+    */
```

Comment:
```suggestion
  /*
   * Generating a random string to use as a document ID, Encryption key, and
   * Encryption keyHint (randomized for demonstration purposes only).
   * These values can be any string, but must be unique.
   */
```
Formatting issue here.